packages/cashier/src/constants/routes-config.tsx

```diff
@@ -62,17 +61,13 @@ const initRoutesConfig = (): TRouteConfig[] => [
             },
             {
                 path: routes.cashier_p2p,
-                component: P2P,
+                component: () => {
+                    window.location.href = 'https://p2p.deriv.com';
```

Comment:
can we check for staging too

Reply:
ah yeah was going to push it, just noticed it as well 👍